dev/ColorPicker/ColorPicker_v1.xaml

```diff
@@ -327,161 +308,280 @@
-                                            </StackPanel>
-                                            <StackPanel Orientation="Horizontal" Margin="0,12,0,0">
-                                                <TextBox x:Name="BlueTextBox" Width="120" MaxLength="3" Text="255"/>
-                                                <TextBlock x:Name="BlueLabel" Text="Blue" VerticalAlignment="Center" Margin="8,0,0,0"/>
-                                            </StackPanel>
-                                        </StackPanel>
-                                        <StackPanel x:Name="HsvPanel" Grid.Row="1" Visibility="Collapsed">
-                                            <StackPanel Orientation="Horizontal" Margin="0,12,0,0">
-                                                <TextBox x:Name="HueTextBox" Width="120" MaxLength="3" Text="0"/>
-                                                <TextBlock x:Name="HueLabel" Text="Hue" VerticalAlignment="Center" Margin="8,0,0,0"/>
-                                            </StackPanel>
-                                            <StackPanel Orientation="Horizontal" Margin="0,12,0,0">
-                                                <TextBox x:Name="SaturationTextBox" Width="120" MaxLength="3" Text="0"/>
-                                                <TextBlock x:Name="SaturationLabel" Text="Saturation" VerticalAlignment="Center" Margin="8,0,0,0"/>
-                                            </StackPanel>
-                                            <StackPanel Orientation="Horizontal" Margin="0,12,0,0">
-                                                <TextBox x:Name="ValueTextBox" Width="120" MaxLength="3" Text="100"/>
-                                                <TextBlock x:Name="ValueLabel" Text="Value" VerticalAlignment="Center" Margin="8,0,0,0"/>
-                                            </StackPanel>
-                                        </StackPanel>
-                                    </StackPanel>
-                                    <StackPanel x:Name="AlphaPanel" Orientation="Horizontal" Margin="0,12,0,0">
-                                        <TextBox x:Name="AlphaTextBox" Width="120" MaxLength="4" Text="100%"/>
-                                        <TextBlock x:Name="AlphaLabel" Text="Opacity" VerticalAlignment="Center" Margin="8,0,0,0"/>
-                                    </StackPanel>
-                                </StackPanel>
-                                <TextBox x:Name="HexTextBox" Grid.Column="1" MaxLength="7" Text="#FFFFFF" Margin="0,12,0,0" Width="132" HorizontalAlignment="Right" VerticalAlignment="Top" />
+
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="{ThemeResource SliderPreContentMargin}" />
+                                    <ColumnDefinition Width="Auto" />
```

Comment:
Please ensure that the v1 styes work as expected. The test coverage for v1 styles is very limited now.

Reply:
Did manual testing with the v1 style and it seemed to work correctly.